### PR TITLE
[JBWS-4333]:Upgrade to WFLY 27.0.0.Final and WFLY 28.0.0.Beta1-SNAPSHOT

### PR DESCRIPTION
**Pull request by jimma**

### CODE REVIEW

**Comment by jbliznak** [comment]
this workaround should be now removed too
https://github.com/jbossws/jbossws-cxf/blob/a71659afdc2d9c13e3f30e453468315135a867b6/modules/testsuite/pom.xml#L613-L625

**Comment by jbliznak** [comment]
hm, the whole profile should be perhaps revisited because of https://github.com/jbossws/jbossws-cxf/pull/246 to get rid of wildfly-webservices-tests-integration

**Comment by jimma** [comment]
> this workaround should be now removed too
> 
> https://github.com/jbossws/jbossws-cxf/blob/a71659afdc2d9c13e3f30e453468315135a867b6/modules/testsuite/pom.xml#L613-L625

Yes, Removed. Thanks for review.

**Comment by jimma** [comment]
> #246

@jbliznak  Yes. Already removed  wildfly-webservices-tests-integration dependency.

**Comment by jimma** [comment]
@jbliznak  Can you please review it again ?

File: pom.xml [hunk 1]
```diff
@@ -1080,14 +1080,14 @@
...
     <!--
-     Name:  wildfly27dev
-     Descr: Test with WFLY27 upstream version -->
+     Name:  wildfly28dev
+     Descr: Test with WFLY28 upstream version -->
     <profile>
-      <id>wildfly27dev</id>
+      <id>wildfly28dev</id>
       <properties>
         <jbossws.integration.target>wildfly2700</jbossws.integration.target>
```
Comment (jbliznak) [inline pom.xml:1088]: what about this value?

Comment (jbliznak) [reply pom.xml:1088]: probably refers to https://github.com/jimma/jbossws-cxf/tree/upgrade-wfly28/modules/resources/src/main/resources/modules

Comment (jimma) [reply pom.xml:1088]: Yes. That's it. We can probably remove this in the future： https://issues.redhat.com/browse/JBWS-4336.

File: modules/testsuite/pom.xml [hunk 2]
```diff
@@ -659,12 +678,13 @@
...
     </profile>
     <!--
-    Descr: Test with the latest WFLY27 upstream version
+    Descr: Test with the latest WFLY28 upstream version
     -->
     <profile>
-      <id>wildfly27dev</id>
+      <id>wildfly28dev</id>
       <properties>
         <additionalGroovyScriptSuffix>-nolegacy</additionalGroovyScriptSuffix>
```
Comment (jbliznak) [inline modules/testsuite/pom.xml:689]: this can be removed, it is not used anywhere anymore

File: pom.xml (hunk 1 above)
Comment (jbliznak) [reply pom.xml:1088]: ok let's ignore that for now

**Comment by jbliznak** [comment]
> @jbliznak Can you please review it again ?

Just one minor unused property and otherwise this looks ok.

**Review by jbliznak — APPROVED** [review]

File: modules/testsuite/pom.xml (hunk 2 above)
Comment (jimma) [reply modules/testsuite/pom.xml]: Fixed. Thanks @jbliznak

**Pull request merged**